packages/react/src/SpaceFinder.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { Fragment, useState } from 'react'
+import { Combobox, Transition } from '@headlessui/react'
+import { CheckIcon, ChevronUpDownIcon } from '@heroicons/react/20/solid'
+
+export function SpaceFinder ({ spaces, selected, setSelected, className }) {
+  const [query, setQuery] = useState('')
+
+  console.log('space', query,  selected)
+
+  const filtered =
+    query === ''
+      ? spaces
+      : spaces.filter((space) =>
+          (space.name() || space.did())
+            .toLowerCase()
+            .replace(/\s+/g, '')
+            .includes(query.toLowerCase().replace(/\s+/g, ''))
+        )
+
+  return (
+    <div className={className}>
+      <Combobox value={selected} onChange={setSelected} by={(a, b) => a.sameAs(b)}>
+        <div className="relative mt-1">
```

Comment:
I'm down to merge this as-is, but just a note that we'll need to translate this Tailwind styling to CSS-variable-based styling before we ship this package for real